aioredlock/algorithm.py

```diff
@@ -81,6 +78,10 @@ def log(self):
         lock_identifier = str(uuid.uuid4())
         error = RuntimeError('Retry count less then one')
 
+        lease_time = self.internal_lock_timeout if lock_timeout == -1 else lock_timeout
+        drift = lease_time * 0.01 + 0.002
+
+
```

Comment:
Empty line, please use linter and autopep8.

Reply:
my bad, it's a rough version, just for u and me to check the implementation, i will update the code